src/Windows/Avalonia.Win32/DxgiSwapchain/DXGI_SWAP_CHAIN_FLAG.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Avalonia.Win32.DxgiSwapchain
+{
+    [Flags]
+    public enum DXGI_SWAP_CHAIN_FLAG
```

Comment:
Ideally all of the new members should be internal. Leaving the minimum (handles?) for the interoperability.

Reply:
> Ideally all of the new members should be internal. Leaving the minimum (handles?) for the interoperability.

Agreed, amended.